dash-pipeline/SAI/utils/dash_p4/dash_p4_table.py

```diff
@@ -419,19 +413,18 @@ def create_sai_attributes(self, sai_api: SaiApi) -> None:
 
             sai_api.attributes.append(counter_attr)
 
-        if self.is_object == "true":
-            # If any match key in this table supports priority, we need to add a priority attribute.
-            if any([key.match_type != "exact" for key in self.keys]) and all(
-                [key.match_type != "lpm" for key in self.keys]
-            ):
-                priority_attr = SaiAttribute(
-                    name=f"SAI_{self.name.upper()}_ATTR_PRIORITY",
-                    description="Rule priority in table",
-                    type="sai_uint32_t",
-                    flags="MANDATORY_ON_CREATE | CREATE_ONLY",
-                )
+        # If any match key in this table supports priority, we need to add a priority attribute.
```

Comment:
this code only covers the attribute part, but not the structure part, hence it results in existing priority entry being removed.

Reply:
Updated. Also cover structure part.